src/Tribe/Models/Post_Types/Venue.php

```diff
@@ -0,0 +1,75 @@
+<?php
+/**
+ * Models an Venue.
+ *
+ * @since   TBD
+ *
+ * @package Tribe\Events\Models\Post_Types
+ */
+namespace Tribe\Events\Models\Post_Types;
+
+use Tribe\Models\Post_Types\Base;
+use Tribe\Utils\Lazy_Collection;
+use Tribe\Utils\Lazy_String;
+use Tribe\Utils\Post_Thumbnail;
+use Tribe__Events__Venue as Venue_Manager;
+use Tribe__Events__Timezones as Timezones;
+use Tribe__Events__Pro__Geo_Loc as Geolocalization;
+
+/**
+ * Class Venue.
+ *
+ * @since   TBD
+ *
+ * @package Tribe\Events\Post_Types
+ */
+class Venue extends Base {
+
+	/**
+	 * {@inheritDoc}
+	 */
+	protected function build_properties( $filter ) {
+		try {
+			$cache_this = $this->get_caching_callback( $filter );
```

Comment:
This is not used anywhere.
If so, it should be removed.
This would also mean that the properties are never cached, this is fine if really there are no properties worth caching.

Reply:
Please, take a look at the templates and add more construct properties, e.g. Venue Details, that it might be worth caching.